internal/util/cephcmds.go

```diff
@@ -219,27 +219,31 @@ func RemoveOMapKey(ctx context.Context, monitors string, cr *Credentials, poolNa
 // CreateObject creates the object name passed in and returns ErrObjectExists if the provided object
 // is already present in rados
 func CreateObject(ctx context.Context, monitors string, cr *Credentials, poolName, namespace, objectName string) error {
-	// Command: "rados <options> create objectName"
-	args := []string{
-		"-m", monitors,
-		"--id", cr.ID,
-		"--keyfile=" + cr.KeyFile,
-		"-c", CephConfigPath,
-		"-p", poolName,
-		"create", objectName,
+	conn := ClusterConnection{}
+	err := conn.Connect(monitors, cr)
+	if err != nil {
+		return err
+	}
+	defer conn.Destroy()
+
+	ioctx, err := conn.GetIoctx(poolName)
```

Comment:
same comment as https://github.com/ceph/ceph-csi/pull/1129#discussion_r434992638

Reply:
I am not sure there is a benefit in caching the IOContext in addition to the ClusterConnection. If you want, open an issue for it so we can investigate and improve that at a later time.